examples/golang/actors/http-hello-world/wit/world.wit

```diff
@@ -0,0 +1,5 @@
+package wasmcloud:hello;
+
+world hello {
+  include wasi:http/proxy@0.2.0-rc-2023-12-05;
```

Comment:
The `proxy` world has a *slightly* different intent -- it's impressive that we can properly implement this, but I think it pulls in stuff you don't need like `outgoing-handler` (since we're not doing any actual proxying yet)

Maybe it's worth considering using logging + http incoming handler?

```wit
    import wasi:logging/logging;
    export wasi:http/incoming-handler@0.2.0-rc-2023-12-05;
```

I mean logging itself also isn't even strictly necessary.

Reply:
Good call! We can just do the handler. I may leave the logging out if wasm time serve doesn't support it

Reply:
Left out logging since `wasmtime serve -Scommon` doesn't include that as an export but I will add in our other docs!